### ISSUE TITLE
Crashes on VS2022 Community - Version 17.7.4

### ISSUE THREAD

**Issue by jds1974** [mattwhitfield/VisualFindReferences#28]
When attempting to use "Visual Find  References...", I get:
![image](https://github.com/mattwhitfield/VisualFindReferences/assets/145588807/43585558-6779-4163-8f36-23117c16d126)

**Comment by mattwhitfield** [comment]
Could you show me what it is that you're clicking on to do that? Or create a small repro?

Thanks

**Comment by jds1974** [comment]
Hi, I think I figured it out, it only seems to 'crash' on VB code, in a VB project. It works fine in C# code/projects. I don't recall seeing anywhere in your docs that it does not work on VB code, unless I missed that?

**Comment by mattwhitfield** [comment]
Oh. Wow. I haven't even considered VB, to be honest. I'll update the docs 👍

Thanks for following up, appreciate it.

**Comment by jds1974** [comment]
Or update the extension so that it does nothing, or the menu options are disabled, when trying to use from a file that's not a .cs? :-)

**Comment by mattwhitfield** [comment]
Yeah, I do a similar thing in Unitverse - I will hide it for VB projects too 👍

**Comment by Alcance-PT** [comment]
Hi,
Can you make it work with VB also? Migrate to C# is very nice, but we have many projects that are on VB, and the work involved does bot allow us to stop using VB...
